### ISSUE TITLE
Can hash function support base64-url-encoding?

### ISSUE THREAD

**Issue by innocentliny** [veler/DevToys#423]
### What's the Problem?

Currently, the hash function only support two output format, one is hex string, another is base64 string.
I need to get base64-url-encoded format to use with HTTP URL.

### Solution/Idea

I hope the output format can add base64-url-encoding.

### Alternatives

Manual or run another tool to make base64-encoded string URL-safe.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |

### DevToys Version

Version 1.0.3.0 | X64 | RELEASE | 7c7c02a | 7c7c02a

**Comment by jwfxpr** [comment]
We are currently discussing changes to the number base converter tool in #384 that would allow you to convert the hash output from hex to base64 using that tool. 😊

**Comment by youkai95** [comment]
@jwfxpr I am not sure if that's correct... That tool will allow you to convert numbers, but not any hex data.

**Comment by jwfxpr** [comment]
@youkai95 all data is a number, if you allow numbers to be big enough. 😊

**Comment by youkai95** [comment]
@jwfxpr I agree, but the tool doesnt support that feature and I am not sure if it could due to everything implied on converting through different bases... I see this more as an encoder/decoder tool
